tests/unit/test_class_subcmd.py

```diff
@@ -106,31 +106,32 @@ class hierarchy defined by `CLASSNAME` argument if provided.
 
   The output format is defined by the output-format general option.
 
-  The includeclassqualifiers, includeclassorigin options define optional
+  The local-only and no-qualifiers options define optional
```

Comment:
Show option names local-only and no-qualifiers with leading dashes.

Reply:
DONE.